### DISCUSSION TITLE
System message in modelSpecs

### DISCUSSION THREAD

**Discussion by mrn-linak** [danny-avila/LibreChat#4075] [Help Wanted]
Modelspecs are a great tool for us to control the roll out of different models, but I haven't found a way to put a system message to these models. Is it correctly understood that Modelspecs don't support system messages? Why is that?

What does other people do, when trying to serve standardised models to a large audience? Modelspecs is the only way that I've found to roll out specific models with custom names, logos, prompts, e.g. to the entire user base.

**Comment by danny-avila** [comment]
`promptPrefix` is the system message

**Reply by mrn-linak** [reply]
Oh, I see! Thanks. I interpreted it as the promptPrefix was prefixed to every message, but instead it is simply the system message? Is that the correct interpretation?
